### PR TITLE
Refactor example and exisiting sdk

### PR DESCRIPTION
**Pull request by shiwangi-shah**
Renaming

example-sdk to runtime-enabled-sdk
existing-sdk to runtime-aware-sdk
for readability. The runtime-enabled sdk is the sdk that is loaded in the sdk runtime process and acts as an ad network or as a mediator.

### CODE REVIEW

File: PrivacySandboxKotlin/runtime-aware-sdk/src/main/java/com/runtimeaware/sdk/FullscreenAd.kt [hunk 1]
```diff
@@ -30,12 +30,12 @@ class FullscreenAd(private val sdkFullscreenAd: FullscreenAd) {
 
     companion object {
         // This method could divert a percentage of requests to a sandboxed SDK and fallback to
-        // existing ad logic. For this example, we send all requests to the sandboxed SDK as long as
+        // existing ad logic. For this runtimeenabled, we send all requests to the sandboxed SDK as long as
```
Comment (notmariazoe) [inline PrivacySandboxKotlin/runtime-aware-sdk/src/main/java/com/runtimeaware/sdk/FullscreenAd.kt:33]: Nit: It seems like this comment description got impacted by the replace. I'd also suggest replacing sandboxed SDKs for something like "in the runtime".

**Review by notmariazoe — CHANGES_REQUESTED** [review]
LGTM. It's just a NIT, but in GH if you make changes after approval, the PR isn't valid anymore :)

File: PrivacySandboxKotlin/runtime-aware-sdk/src/main/java/com/runtimeaware/sdk/FullscreenAd.kt (hunk 1 above)
Comment (shiwangi-shah) [reply PrivacySandboxKotlin/runtime-aware-sdk/src/main/java/com/runtimeaware/sdk/FullscreenAd.kt]: Fixed

**Review by notmariazoe — APPROVED** [review]
LGTM, thanks

**Pull request merged**